internal/testutil/testutil.go

```diff
@@ -29,9 +29,6 @@ func (v Version) K8s() (K8s, error) {
 // - REMEMBER TO ADD IT TO AllVersions() BELOW.
 
 const (
-	Testdata116 = "1_16"
-	Testdata117 = "1_17"
```

Comment:
since we are still supporting those versions why do we need to remove the test data?

Reply:
I had issues to launch local kubernetes clusters for those versions using a modern minikube in arm. We could dig into it or we could keep the data we had and add exceptions for the _horizontalpodautoscaler_ metrics, but I am not sure if maintaining the static data for those versions (which are [not supported by kubernetes](https://kubernetes.io/releases/patch-releases/#non-active-branch-history)) is useful.